tests/node-redis-dockerfile

```diff
@@ -0,0 +1,21 @@
+# syntax=docker/dockerfile:1
+
+FROM node:18.7.0
+ENV NODE_ENV=development
+
+WORKDIR /app
+# Clone node-redis dragonfly fork
+RUN git clone https://github.com/dragonflydb/node-redis.git
```

Comment:
`git clone -b branch` should save you the "switch" command below

Reply:
done